### DISCUSSION TITLE
I cant use IDE to execute javascript

### DISCUSSION THREAD

**Discussion by Lil-Maka** [community/community#136727] [Programming Help] [answered]
### Body

Hello i want to use node + my code in order to execute some javascript code, but it says i have an error when execute code, i have node installed what can i do? I use vscode

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by andresito1969** [comment, marked as answer]
Hello, what you can do is install code runner in extensions of your vscode, hope it helps!
